src/libs/actions/Policy.ts

```diff
@@ -1041,45 +1104,39 @@ function generateDefaultWorkspaceName(email = '') {
         defaultWorkspaceName = 'My Group Workspace';
     }
 
-    if (allPolicies.length === 0) {
+    if (!isNotEmptyObject(allPolicies ?? {})) {
```

Comment:
Please make sure check works the same way as before

Reply:
The double negation made it quite hard to read so I exchanged the function for `isEmptyObject` which is equal to the code before: `return Object.keys(obj ?? {}).length === 0;` and it automatically defaults to an empty object so I also removed that. Current change works exactly like before.